bubblewrap.c

```diff
@@ -926,6 +934,25 @@ parse_args (int *argcp,
           argv += 1;
           argc -= 1;
         }
+      else if (strcmp (arg, "--exec-label") == 0)
+        {
+          if (argc < 2)
+            die ("--exec-label takes an argument");
+
+          opt_exec_label = argv[1];
+          argv += 1;
+          argc -= 1;
+        }
+      else if (strcmp (arg, "--file-label") == 0)
+        {
+          if (argc < 2)
+            die ("--file-label takes an argument");
+
+          opt_file_label = argv[1];
+          label_create_file(opt_file_label);
```

Comment:
Personally, I would inline this here since we're only calling it once.  Also, potentially error out `#ifndef HAVE_SELINUX`?